src/Confluent.Kafka/Confluent.Kafka.csproj

```diff
@@ -1,38 +1,40 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <ProjectTypeGuids>{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
-    <Authors>Confluent Inc.;Andreas Heider</Authors>
-    <Description>Confluent's .NET Client for Apache Kafka</Description>
-    <Copyright>Copyright 2016-2020 Confluent Inc., Andreas Heider</Copyright>
-    <PackageProjectUrl>https://github.com/confluentinc/confluent-kafka-dotnet/</PackageProjectUrl>
-    <PackageLicenseUrl>https://github.com/confluentinc/confluent-kafka-dotnet/blob/master/LICENSE</PackageLicenseUrl>
-    <PackageIconUrl>https://raw.githubusercontent.com/confluentinc/confluent-kafka-dotnet/master/confluent-logo.png</PackageIconUrl>
-    <PackageReleaseNotes>https://github.com/confluentinc/confluent-kafka-dotnet/releases</PackageReleaseNotes>
-    <PackageTags>Kafka;Confluent;librdkafka</PackageTags>
-    <PackageId>Confluent.Kafka</PackageId>
-    <Title>Confluent.Kafka</Title>
-    <AssemblyName>Confluent.Kafka</AssemblyName>
-    <VersionPrefix>2.3.0</VersionPrefix>
-    <TargetFrameworks>netstandard2.0;netstandard1.3;net462;net6.0</TargetFrameworks>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>Confluent.Kafka.snk</AssemblyOriginatorKeyFile>
+	<ProjectTypeGuids>{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+	<Authors>Confluent Inc.;Andreas Heider</Authors>
+	<Description>Confluent's .NET Client for Apache Kafka</Description>
+	<Copyright>Copyright 2016-2020 Confluent Inc., Andreas Heider</Copyright>
+	<PackageProjectUrl>https://github.com/confluentinc/confluent-kafka-dotnet/</PackageProjectUrl>
+	<PackageLicenceExpression>Apache-2.0</PackageLicenceExpression>
+	<RepositoryUrl>https://github.com/confluentinc/confluent-kafka-dotnet/</RepositoryUrl>
```

Comment:
Can we update it to `https://github.com/confluentinc/confluent-kafka-dotnet.git` everywhere